### PR TITLE
Adding Sorted Sets

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**

### CODE REVIEW

File: docs/develop/api-reference/sorted-set-collections.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+- If the set does not exist, this method creates a new sorted set item with the element(s) passed in.
+
+- If the set exists, for each [SortedSetElement](#sortedsetelement) in the array, each element is added to the sorted set if that **value** doesn't exist. If the value of that element does exist, that element is overwritten.
+
+| Name            | Type               | Description                                   |
+| --------------- | ------------------ | --------------------------------------------- |
+| cacheName       | String             | Name of the cache.                            |
+| setName         | String             | Name of the sorted set item to be altered. |
+| elements        | [SortedSetElement](#sortedsetelement)[] | Elements to be added to the sorted set by this operation. |
+| ttl             | [CollectionTTL object](./collection-ttl.md) | TTL for the set item in cache. This TTL takes precedence over the TTL used when initializing a cache connection client. |
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:27]: Need to update to "TTL for the **sorted** set item"

**Review by cprice404 — APPROVED** [review]
hero

File: docs/develop/api-reference/sorted-set-collections.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+Removes an element from a sorted set.
+
+| Name            | Type             | Description                                   |
+| --------------- | ---------------- | --------------------------------------------- |
+| cacheName       | String           | Name of the cache.                            |
+| setName         | String           | Name of the set item to be altered. |
+| elements        | All \| string[] \| bytes[] | Element to be removed by this operation. |
+
+You can remove either elements all elements or a set of elements.
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:97]: This wording doesn't make sense. Too many "elements" in there

File: docs/develop/api-reference/sorted-set-collections.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+</details>
+
+### SortedSetGetRank
+
+What position is this element in the specificed sorted set?
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the set item to be altered. |
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:116]: This description is not right, should be `Name of the sorted set item.`

File: docs/develop/api-reference/sorted-set-collections.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+| element         | String \| bytes | Element to be removed by this operation. |
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:117]: This description isn't right, should be `Name of the element to retrieve rank` or something similar

File: docs/develop/api-reference/sorted-set-collections.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+:::note
+
+The resulting incremented score must be between -9223372036854775808 and 9223372036854775807, ie. a signed double 64-bit float. If not, there will be an error response.
+
+:::
+
+Examples:
+
+- When the element does not exist in the sorted set, `SortedSetIncrementScore(cache, cacheName, setName, value, 10)` will set the element's score to 10.
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:143]: The code has too many arguments. Need to drop the first one: `cache`

File: docs/develop/api-reference/sorted-set-collections.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+- When the existing element is a value:score of "{ 'KesselRun' : 12 }" , `SortedSetIncrementScore(cacheName, setName, value, 10)` will set the element's score to 22.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the sorted set item to be altered. |
+| value           | String \| bytes | Value for the element to be incremented by this operation. |
+| amount          | Number          | The quantity to add to the score. May be positive, negative, or zero. Defaults to 1. |          
+| ttl             | [CollectionTTL object](./collection-ttl.md) | This will come back as a TTL construct. |
```
Comment (allenheltondev) [inline docs/develop/api-reference/sorted-set-collections.md:152]: I'm thinking the description of the ttl should be consistent with what you already have on this page `TTL for the set item in cache. This TTL takes precedence over the TTL used when initializing a cache connection client.`

**Review by allenheltondev — CHANGES_REQUESTED** [review]

**Pull request merged**